src/cbmc_starter_kit/template-for-repository/proofs/Makefile.common

```diff
@@ -682,78 +685,100 @@ else
 	  --description "$(PROOF_UID): setting static variables to nondet"
 endif
 
-# Omit unused functions (sharpens coverage calculations)
+# Link CPROVER library if DFCC mode is on
 $(HARNESS_GOTO)4.goto: $(HARNESS_GOTO)3.goto
-ifneq ($(strip $(DFCC_MODE)),)
+ifneq ($(strip $(USE_DYNAMIC_FRAMES)),)
 	$(LITANI) add-job \
-	  --command 'cp $^ $@' \
+	  --command \
+	    '$(GOTO_INSTRUMENT) $(CBMC_VERBOSITY) $(ADD_LIBRARY_FLAG) $(CBMC_OPT_CONFIG_LIBRARY) $^ $@' \
```

Comment:
yes they are still going to work as intended. When using contracts for function pointers, function pointer restrictions are expected to contain the `function_pointer -> contract_function` mapping explicitly. This will have to be changed later when function pointers get resolved during symex.